jsonnet/components/thanos-querier.libsonnet

```diff
@@ -144,7 +144,7 @@ function(params)
               },
             },
             resourceAttributes: {
-              apiVersion: 'metrics.k8s.io/v1beta1',
+              apiVersion: 'v1',
```

Comment:
Verified that `apiVersion` is a supported field: https://github.com/brancz/kube-rbac-proxy/blob/master/examples/resource-attributes/README.md?plain=1#L74.